src/functionalTest/resources/request/casedata/ccd-callback-caseworker-issue-application-about-to-submit.json

```diff
@@ -90,18 +90,17 @@
     "Fees": null,
     "PaymentTotal": "55000"
   },
-  "payments": [
+  "applicationPayments": [
     {
       "id": null,
       "value": {
-        "paymentDate": null,
-        "paymentFeeId": "FEE0001",
-        "paymentAmount": 55000,
-        "paymentSiteId": "AA04",
-        "paymentStatus": "success",
-        "paymentChannel": "online",
-        "paymentReference": "paymentRef",
-        "paymentTransactionId": "ge7po9h5bhbtbd466424src9tk"
+        "created": null,
+        "feeCode": "FEE0001",
+        "amount": 55000,
+        "status": "success",
+        "channel": "online",
+        "reference": "paymentRef",
+        "transactionId": "ge7po9h5bhbtbd466424src9tk"
```

Comment:
do we not need site id ? Also is the request body as per new payment api format ?

Reply:
Site ID has been removed by the payments team, I have a separate PR for the work on the frontend. This is the new structure for payments. I've also added an `updated` datetime but not used it in any tests